### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by rsenwar**

### CODE REVIEW

File: invoice2data/main.py [hunk 1]
```diff
@@ -7,15 +7,15 @@
 from os.path import join
 import logging
 
-from .input import pdftotext
-from .input import pdfminer
-from .input import tesseract
+from input import pdftotext
+from input import pdfminer
+from input import tesseract
```
Comment (duskybomb) [inline invoice2data/main.py]: Have you tested this commit @rsenwar ?
Doesn't seem to work in my environment.

Comment (m3nu) [reply invoice2data/main.py]: Agree. This won't work. I just undid those changes earlier.

Comment (rsenwar) [reply invoice2data/main.py]: I have tested it, works fine.
I get an error without these changes.

'from .input import pdftotext
ValueError: Attempted relative import in non-package'

Comment (rsenwar) [reply invoice2data/main.py]: Yeah, I understood, I have run that as a whole package.

Comment (m3nu) [reply invoice2data/main.py]: Are you running the main-file directly? That won't work and shouldn't work. Needs to run as module. See [here](https://stackoverflow.com/a/28154841) for a very exhaustive explanation.

tl;dr:
- They recommend absolut imports. To get those you still need to add the package name.
- Many people including me like relative imports, prefixed with a dot. The only downside is that you can't run a file directly. See *solution 1* in the link above.

Comment (m3nu) [reply invoice2data/main.py]: So the way to run it is either:

- `python -m invoice2data.main -h`
- `invoice2data` (linked cli tool)

This won't work:
- `python invoice2data/main.py`

Comment (rsenwar) [reply invoice2data/main.py]: My bad! Yeah It works fine.

Comment (m3nu) [reply invoice2data/main.py]: Great to hear. We're all here to learn. I also had to look up the precise reason. I guess some years ago I just got used to relative imports and ran every bigger project as module. Then forgot  the exact reason.
